docs/getting-started.md

```diff
@@ -161,76 +175,114 @@ First we need to setup CLion
+
 1. Open VSCode
 2. Navigate to `Software/src/software/geom/BUILD`
 3. On top of every `cc_test`, `cc_library` and `cc_binary` there should be a `Test ...`, `Build ...` or `Run ...` for the respective target.
 4. Click `Test //software/geom:angle_test` to run the `angle_test`
 
 ### Running our AI, Simulator, SimulatedTests or Robot Diagnostics
 
-1. Open your terminal, `cd` into `Software/src` and run `ifconfig`.
-2. Pick the network interface you would like to use:
-    1. If you are running things locally, you can pick any interface that is not `lo`
-    2. If you would like to communicate with robots on the network, make sure to select the interface that is connected to the same network as the robots.
-3. Run our AI: `bazel run //software:full_system -- --interface=[interface_here] --backend=SimulatorBackend`
-    - This will launch the Visualizer, which displays what the AI is currently "seeing" and allows us to interact with the AI through the dynamic parameters.
-    - The field should be empty, as we are currently not receiving SSL Vision packets.
-4. Run our Simulator: `bazel run //software:standalone_simulator_main -- --interface=[interface_here]`
-    - The Simulator runs our firmware and Box2D (a physics engine) to simulate how our robots would behave on the field.
-    - The Simulator outputs SSL Vision packets, which contain position information of all robots and the ball.
-    - Our AI can now "see" the robots, and they should be displayed on the Visualizer.
-    - You can use ctrl-click to move the ball around in the Simulator, and try changing the Play Override on the Visualizer to see the robots move!
-5. Run Robot Diagnostics: `bazel run //software:robot_diagnostics:robot_diagnostics_main -- --interface=[interface_here] --backend=WifiBackend`
+1. Run our AI on Thunderscope:
+    - Thunderscope is the software that coordinates our AI, Simulator, Visualizer and RobotDiagnostics
+    - After launching Thunderscope, we can see what the AI is currently "seeing" and interact with it through dynamic parameters. 
+    - If we want to run with simulated AI vs AI:
+        - `./tbots.py run thunderscope_main` will start Thunderscope with our Simulator, a blue FullSystem and yellow FullSystem. Each FullSystem contains the respective AI for each side. The command will start Thunderscope and set up communication between the Simulator, GameController and FullSystems.
+        - We use ER Force's Simulator to simulate how our robots would behave on the field. This simulator is powerful because it includes vision noise, allowing us to further stress test our gameplay.
+        - The Simulator outputs SSL Vision packets, which contain position information of all robots and the ball.
+        - Our AI can now "see" the robots, and they should be displayed on the Visualizer.
+        - You can use ctrl-click to move the ball around in the Simulator, and try changing the Play Override on the Visualizer to see the robots move!
```

Comment:
I think it could be helpful if we add to the doc that they will need to be change the game state through the game controller, and maybe give an example of what gamestate they should try (e.g. STOP > Kick off > Normal Start)

Reply:
done